async_firebase/client.py

```diff
@@ -421,55 +390,65 @@ async def push(  # pylint: disable=too-many-locals
 
     async def push_multicast(
         self,
-        device_tokens: t.Union[t.List[str], t.Tuple[str]],
+        multicast_message: MulticastMessage,
         *,
-        android: t.Optional[AndroidConfig] = None,
-        apns: t.Optional[APNSConfig] = None,
-        data: t.Optional[t.Dict[str, str]] = None,
-        notification: t.Optional[Notification] = None,
-        webpush: t.Optional[WebpushConfig] = None,
         dry_run: bool = False,
-    ) -> FcmPushMulticastResponse:
+    ) -> FcmPushBatchResponse:
```

Comment:
So, at first glance, this method has `multicast` in its name, `multicast` in the name of one of its parameters, but returns `FcmPushBatchResponse`.

`FcmPushMulticastResponse ` is renamed to `FcmPushBatchResponse`, but now fits well only the other method `push_batch`. Just curious, if we could come up with a general name then that fits both methods to not confuse our end users?

Reply:
@ndmytro In my opinion, `FcmPushBatchResponse` covers both scenarios reasonably well and it's pretty general compared to `FcmPushMulticastResponse`.  I don't think it is worth looking for a better name.